### ISSUE TITLE
Logginglevel INFO in ExpressionThreadPoolExecutor

### ISSUE THREAD

**Issue by cweitkamp** [eclipse/smarthome#3769]
After upgrading my productive environment to OH 2.1 yesterday I see a lot of lines from ExpressionThreadPoolExecutor in my log. They are somehow related to the astro binding, but question is if logging level INFO is the right choice here.
```
2017-06-29 00:00:00.333 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 0 0 29 6 ? 2017' has no future executions anymore
2017-06-29 03:01:00.538 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 1 3 29 6 ? 2017' has no future executions anymore
2017-06-29 03:01:00.555 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 1 3 29 6 ? 2017' has no future executions anymore
2017-06-29 03:01:00.567 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 1 3 29 6 ? 2017' has no future executions anymore
...
2017-06-29 04:19:00.476 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 19 4 29 6 ? 2017' has no future executions anymore
2017-06-29 04:19:00.532 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 19 4 29 6 ? 2017' has no future executions anymore
...
2017-06-29 05:08:00.124 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 8 5 29 6 ? 2017' has no future executions anymore
2017-06-29 05:13:00.120 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 13 5 29 6 ? 2017' has no future executions anymore
2017-06-29 05:13:00.124 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 13 5 29 6 ? 2017' has no future executions anymore
...
2017-06-29 07:00:00.300 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 0 7 29 6 ? 2017' has no future executions anymore
2017-06-29 07:00:00.303 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 0 7 29 6 ? 2017' has no future executions anymore
2017-06-29 07:00:00.349 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 0 7 29 6 ? 2017' has no future executions anymore
2017-06-29 07:00:00.359 [INFO ] [Manager$ExpressionThreadPoolExecutor] - Expression '0 0 7 29 6 ? 2017' has no future executions anymore
```

**Comment by sjka** [comment]
I like your way of thinking! 😄 
#3751, i.e. https://github.com/eclipse/smarthome/pull/3751/files#diff-1c9d01161e0b1617c2e15e351b0ff946R251

**Comment by cweitkamp** [comment]
Me, too. :wink:
Didn't see your PR, only looked into the issues.

**Comment by cweitkamp** [comment]
Closed with PR #3751.
